skins/oasis/css/core/responsive-core.scss

```diff
@@ -230,7 +230,7 @@
 			width: auto;
 		}
 
-		.global-notification {
+		.banner-notification {
 			width: 100%;
```

Comment:
This is causing some funky behavior on small screens. 
![screen shot 2015-02-19 at 6 10 02 pm](https://cloud.githubusercontent.com/assets/201367/6280488/ed9bbc0e-b862-11e4-922f-e261b74b292b.png)